installation-and-setup/offline-mode.md

```diff
@@ -0,0 +1,59 @@
+
+## Prerequisites
+
+- Requires licensed seat under the **Fiddler Everywhere Enterprise** subscription tier.
+- Supported on Windows for both physical and virtual work machines.
+- (Windows only) Requires Active Directory configuration with valid AD user (AD principal email).
+- Supported on macOS with limited support for JAMF users only.
+
+## Configuration Instructions
+
+Execute the following steps to enable the offline mode:
+
+1. Start Fiddler Everywhere on the device with limited internet access.
+
+2. Upon login attempt, the Fiddler Everywhere application will display the "Something Went Wrong" screen. From that same screen, copy the information from the text field that contains details about the operating system and the user's principal email.
+
+!["Something Went Wrong" screen on device with limited internet access](../images/offline/offline-mode-something-went-wrong-screen.png)
+
+3. Send the above information to the support team alongside your [offline license request](#request-an-offline-license).
+
+Once the request is received and approved, our team will reach out and send an individual offline license file that you must install on your device as follows:
+
+- Download and install [the latest version of Fiddler Everywhere](https://www.telerik.com/download/fiddler-everywhere).
+
+- Ensure that the Fiddler Everywhere application is closed.
+
+- Open the archive **license.zip** (which you will receive from the support team) and extract the **license.lic** file. You will need only this file, not the archive itself.
+
+    * Instructions for **Windows** - Create and navigate to the following folder on your Windows PC/Server.
+
+    ```sh
+        %PROGRAMDATA%/Fiddler/License
+    ```
+
+    * Instructions for **macOS** - Create and navigate to the following folder on your macOS device.
+
+    ```sh
+        ~/Library/Application Support/Fiddler Everywhere/License
+    ```
+
+- Inside the **License** folder, place the **license.lic** file. Ensure the file has the name **license.lic**.
+
+Finally, restart Fiddler Everywhere to use the offline mode for the currently logged user. Note that the logged user must be identical to the one specified as the user's principal email in your support request.
+
+## Request an Offline license
+
+The offline mode is explicitly enabled through an individual offline license file. To request your license file, contact the Fiddler Everywhere technical support team through the [Telerik's Support Center](https://www.telerik.com/account/support-center
+) or contact us directly at [fiddler-support@progress.com](fiddler-support@progress.com). Your request should include the information from the second step in the [configuration instructions](#configuration-instructions) and the licensed user that holds a seat for the **Fiddler Everywhere Enterprise** subscription tier.
```

Comment:
"Your request should include the information from the second step in the [configuration instructions](https://github.com/telerik/fiddler-everywhere-docs/diffs/1?base_sha=3911b8fae3fbbff4e1832b2938453ac328f4833d&branch=niliev%2Foffline&commentable=true&name=niliev%2Foffline&pull_number=555&qualified_name=refs%2Fheads%2Fniliev%2Foffline&sha1=3911b8fae3fbbff4e1832b2938453ac328f4833d&sha2=125b4e899be8c4c2700ba1f2b1ae2a73113ae104&short_path=158cecd&unchanged=expanded&w=false#configuration-instructions) and the licensed user that holds a seat for the Fiddler Everywhere Enterprise subscription tier."

I think you can improve this sentence. Right now I don't understand what information about the licensed user is needed. Is it the email of the licensed user, the one from their Telerik account, or something else?